index.js

```diff
@@ -189,7 +189,11 @@ exports.evm2wast = function (evmCode, opts = {
     if (opts.chargePerOp) {
       segment += `(call $useGas (i64.const ${op.fee})) `
     }
-    gasCount += op.fee
+    // do not charge gas for interface methods
+    // TODO: implement proper gas charging and enable this here
+    if (opint < 0x30 || (opint > 0x45 && opint < 0xa0)) {
```

Comment:
would be better to have a for explicit list of opcodes which translate to interface methods and opcodes that don't

Reply:
relevant doc is here

Reply:
Yes, that is an option which came up (add a field to `opcodes.js`), but this was a quick and dirty way to progress.